control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -175,7 +196,9 @@ func (r *Reconciler) reconcileKind(ctx context.Context, channel *messagingv1beta
 	}
 	coreconfig.SetDeadLetterSinkURIFromEgressConfig(&channel.Status.DeliveryStatus, channelResource.EgressConfig)
 
-	// Update contract data with the new contract configuration
+	r.reconcileSubscribers(ctx, kafkaClient, kafkaClusterAdmin, channel, channelResource)
```

Comment:
I'd expect this method call to return an error.

When there is an error, we still want to update the CM but then `ReconcileKind` should return an error so that we retry later.

Reply:
Good point. Done